packages/generator-ui5-react/src/app/templates/README.md

```diff
@@ -23,24 +45,52 @@ REACT_APP_PROXY_SAP_USERNAME=YOUR_USER
 REACT_APP_PROXY_SAP_PASSWORD=YOUR_PASSWORD
 ```
 
-## Available Scripts
+### Deployment
+
+Use the file `.ui5deployrc` to configure your deployment.
+
+### OData Services
 
-In the project directory, you can run:
+We generate an OData client out of provided metadata and some configuration
+by using [odata2model](https://www.npmjs.com/package/@odata2ts/odata2model).
 
-### `npm run start`
+Configure the generation in `odata2ts.config.js`.
+
+As an example, the [Trippin OData V4 example service](https://www.odata.org/odata-services/)
+is integrated by default. To provide your own service download the metadata (BASE_URL/$metadata)
+and overwrite `src/asset/odata/odata-service.xml`.
+
+Then generate a type-safe OData client by `npm run gen-odata`. Files will be generated under
+`src/odata/odata-service`.
+
+## Development: `npm run start`
 
 Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Automatically opens [http://localhost:3000](http://localhost:3000) in your main browser.
+Deactivate this behavior via `BROWSER=none` in your `.env.local` file.
 
 The page will reload if you make edits.\
-You will also see any lint errors in the console.
+You will also see any lint and typescript errors in the console.
 
-### `npm run test`
+### Testing: `npm run test`
 
 Launches the test runner in the interactive watch mode.\
 See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+To create a test for file `ExampleScreen.ts`, create a file with the suffix `.test.ts` right beside the file under test:
+`ExampleScreen.test.ts`. Base file content:
```

Comment:
Does it make sense to add a note that all test files must be inside `src`?

Reply:
Added the note about the ´src´ folder.